traefik-mixin/config.libsonnet

```
@@ -0,0 +1,26 @@
+{
+  _config+:: {
+    // alerts thresholds
+    traefik_tls_expiry_days_critical: 7,
+    traefik_tls_expiry_days_warning: 14,
+    timeSeriesLabels: '',
```

```suggestion
    filteringSelector: '',
```

By convention, we call this `filteringSelector` in this mixin collection, a set of static filters that should be included in all alerts and dashboard queries.

---

agreed, changed 13e9ae408f233b169540ae4dda10f1f1b1fde817